pkg/metrics/metrics.jsx

```diff
@@ -307,6 +307,9 @@ class CurrentMetrics extends React.Component {
                      * which are always at 100% capacity, but are uninteresting for disk usage alerts */
                     if ((fields[0].indexOf("/loop") >= 0 && options.indexOf('ro') >= 0))
                         result.add(fields[1]);
+                    /* hide flatpaks */
+                    if ((fields[0].indexOf('revokefs-fuse') >= 0 && fields[1].indexOf('flatpak') >= 0))
+                        result.add(fields[1]);
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17761-20220922-163244-044cdcae-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
Can you please add a check for this to testDiskIO, similar to the "does not show read-only devices"?
```py
        # read-only loop devices are not shown
        self.assertFalse(b.is_present(".pf-c-progress[data-disk-usage-target='/var/cockpit-ro-test']"))
```

i.e. mount something which looks like a flatpak.

Reply:
This would require flatpak to be installed on the test image, as revokefs-fuse is part of the flatpak package. So we would either lax the check to include "fuse" or skip it.

Reply:
Oh, because that "revokefs-fuse" file system isn't available on our VMs, I see. I do have flatpak installed, and even running (Signal app), and even my system doesn't have that file system.

Reply:
I think it's due to some portal handling which fails or isn't cleaned up, which makes me reconsider how useful this change is...

Reply:
Altho.. I have it now, seems updating stuff also causes these files

![image](https://user-images.githubusercontent.com/67428/192525724-2f2d9424-1d52-49f2-87d4-24a3aee0bc39.png)